core/src/main/java/com/dtp/core/thread/DtpExecutor.java

```diff
@@ -149,47 +146,25 @@ public void execute(Runnable task, long startTimeout) {
 
     @Override
     public void execute(Runnable command) {
-        super.execute(wrapTasks(command));
+        DtpRunnable dtpRunnable = (DtpRunnable) wrapTasks(command);
+        dtpRunnable.startTimeoutTask(this, QUEUE_TIMEOUT, queueTimeout, queueTimeoutCount);
```

Comment:
Both queueTimeout and queueTimeoutCount can be obtained from the this object, so only passing the this parameter is sufficient.